jobsdb/jobsdb.go

```diff
@@ -770,6 +798,10 @@ func (jd *Handle) init() {
 		jd.config = config.Default
 	}
 
+	if jd.conf.payloadColumnType == 0 {
+		jd.conf.payloadColumnType = payloadColumnType(jd.config.GetIntVar(int(TEXT), 1, "JobsDB.payloadColumnType"))
+	}
```

Comment:
yeah will make the config change, makes sense
as for the default - I've just used the new TEXT column because it's faster..
No reason to not use it right..? We'll have to change all environments to TEXT at some point.